components/Tabs/Tabs.js

```diff
@@ -1,61 +1,76 @@
 class TabLink {
   constructor(tabElement){
     // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
+    this.tabElement = tabElement;    
     // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
-    
+    this.tabData = tabElement.dataset.tab; 
     // We need to find out if a user clicked 'all' cards or a specific category.  Follow the instructions below to accomplish this task:    
     
-    /* <- Delete this comment block when you work on the if statement
+    //  <- Delete this comment block when you work on the if statement
     // Check to see if this.tabData is equal to 'all'
-    if(){
-      // If `all` is true, select all cards regardless of their data attribute values
-      // this.cards = ;
-    } else {
-      // else if `all` is false, only select the cards with matching this.tabData values
-      // this.cards = ;
-    }
-    /* <- Delete this comment block when you work on the if statement
+    if(this.tabData === 'all'){
+        // If `all` is true, select all cards regardless of their data attribute values
+        this.cards = document.querySelectorAll(".card");
+        console.log(this.cards);
+      } else {
+        // else if `all` is false, only select the cards with matching this.tabData values
+        this.cards = document.querySelectorAll(`div.card[data-tab="${this.tabData}"]`);
+      }
```

Comment:
If you wanted to make this a little more dry, you could convert this into a ternary statement. ie.
```js
this.cards = this.tabData === 'all'
  ? document.querySelectorAll(".card")
  : document.querySelectorAll(`div.card[data-tab="${this.tabData}"]`);
```